artifacts/kernel.json

```diff
@@ -0,0 +1,8 @@
+{
+  "argv": [
+    "kubeflow-kernel",
+    "{connection_file}"
+  ],
+  "display_name": "Kubeflow",
+  "language": "Python"
```

Comment:
Python3 vs Python2?  we probably need to make clear (even though the pr doesn't implement python integration)